app/routes/circulars/index.tsx

```diff
@@ -151,12 +183,18 @@ export default function () {
   const allItems = [...(newItem ? [newItem] : []), ...(items || [])]
 
   const [searchParams] = useSearchParams()
+  console.log('default function search params', searchParams.toString())
   const query = searchParams.get('query') ?? undefined
+  // const startDate = searchParams.get('startDate') ?? undefined
+  // const endDate = searchParams.get('endDate') ?? undefined
+
   let searchParamsString = searchParams.toString()
   if (searchParamsString) searchParamsString = `?${searchParamsString}`
 
-  const [input, setInput] = useState(query)
-  const clean = input === query
+  const [inputQuery, setInputQuery] = useState(query)
+  // const [inputDateGte, setInputDateGte] = useState(startDate)
+  // const [inputDateLte, setInputDateLte] = useState(endDate)
+  const clean = inputQuery === query //&& inputDateGte === startDate && inputDateLte === endDate
```

Comment:
These commented out lines can be deleted

Reply:
these are for the frontend, but yeah, probably best to leave them out until the actual frontend PR